### PR TITLE
toastify installed

### PR DESCRIPTION
**Pull request by RDiMag**
I have installed Toastify and added the ToastMessage.tsx file. I'm not sure how to do further setup or what kind of error messages to add.

### CODE REVIEW

File: src/components/ToastMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+function App() {
```
Comment (jpatrici) [inline src/components/ToastMessage.tsx:6]:
```suggestion
function ToastMessage() {
```

**Review by jpatrici — COMMENTED** [review]
the classic package lock json is back, can you please update and make sure it doesnt go back to v1?

File: src/components/ToastMessage.tsx (hunk 1 above)
Comment (RDiMag) [reply src/components/ToastMessage.tsx:6]: Thanks! I made this change in the second branch I just created.

**Comment by RDiMag** [comment]
Closing as package-lock was getting corrupted by my older version of npm.

**Pull request closed without merging**